### ISSUE TITLE
Use friendly name instead of appid

### ISSUE THREAD

**Issue by wulftone** [nathan-v/aws_okta_keyman#79]
It would be handy to use the "friendly name" instead of the app-id for logging in.  Right now I have to do
```bash
aws_okta_keyman --appid xxxxxxxxxxxx/yyy
```
where it would be much nicer to be able to do
```
aws_okta_keyman --friendlyname staging-env
```

**Comment by nathan-v** [comment]
Hi @wulftone. Have you tried using the tool without the appid sepecified? You'll get a list of AWS accounts to pick from that will have their friendly names (from Okta) displayed.

**Comment by wulftone** [comment]
Yes I saw that, I am looking for a noninteractive way to specify the choice, though. The appid works, but it is less good from a code maintenance perspective since it's not human-readable.

**Comment by wulftone** [comment]
Just wanted to mention I tried the `--name` option and it didn't work for this purpose.  I was still prompted to pick an account. :(
